### DISCUSSION TITLE
Context deadline exceeded : Convert HTML to PDF

### DISCUSSION THREAD

**Discussion by asuwini** [gotenberg/gotenberg#958] [Q&A] [answered]
I’m able to generate PDFs from HTML code up to approximately 12 MB, but I encounter an error when attempting to generate larger files. The error message is:

"convert HTML to PDF: convert to PDF: context deadline exceeded."

Please help me to resolve this issue

**Comment by gulien** [comment, marked as answer]
Hello @asuwini 👋 

Please check: https://gotenberg.dev/docs/troubleshooting#timeouts-chromium

**Reply by asuwini** [reply]
Thank you @gulien ,

I have increased the api timeout duration to 1200sec.
